examples/nodejs/storage-load-gen/README.md

```diff
@@ -0,0 +1,117 @@
+<img src="https://docs.momentohq.com/img/momento-logo-forest.svg" alt="logo" width="400"/>
```

Comment:
technically you don't need to commit this file since it will get overwritten. not a big deal though, just commenting in case anyone else is reviewing